.github/workflows/hosted-pure-workflow.yml

```diff
@@ -96,6 +88,13 @@ jobs:
         run: |
           cmake --preset ninja-multi-vcpkg
 
+      - name: check format
+        uses: jidicula/clang-format-action@v4.14.0
+        if: runner.os == 'Linux'
```

Comment:
is this working only on Linux? Not too bad, but a bit unfortunate.

Reply:
Unfortunately setting up clang-format on windows and linux is a bit more difficult.

Reply:
got it, thank you